src/components/styles/collapsed.module.css

```diff
@@ -0,0 +1,28 @@
+@value colors: "./tokens/colors.module.css";
+@value color-box-shadow, color-border, color-background-primary from colors;
+
+.container {
+  display: inline-flex;
+  flex-direction: column;
+  padding: 10px;
+  border: 1px solid color-border;
+  box-shadow: 0 7px 9px color-box-shadow;
+  border-radius: 10px;
+  font-family: system-ui;
+  bottom: 10px;
+  right: 10px;
+  position: fixed;
+  z-index: 2147483647;
```

Comment:
magic number

Reply:
It's maximum of common browsers https://stackoverflow.com/questions/491052/minimum-and-maximum-value-of-z-index
Users can change this z-index with using classes